apps/io-session-manager-internal/src/controllers/info.ts

```diff
@@ -2,20 +2,26 @@ import { httpAzureFunction } from "@pagopa/handler-kit-azure-func";
 import * as H from "@pagopa/handler-kit";
 import * as RTE from "fp-ts/ReaderTaskEither";
 
-import {
-  getCurrentBackendVersion,
-  getValueFromPackageJson,
-} from "../utils/package";
+import { pipe } from "fp-ts/lib/function";
+import { ApplicationInfo } from "../generated/definitions/internal/ApplicationInfo";
+import { InfoService, InfoServiceDeps } from "../services/info";
+
+type Dependencies = {
+  InfoService: InfoService;
+} & InfoServiceDeps;
 
 export const makeInfoHandler: H.Handler<
   H.HttpRequest,
-  H.HttpResponse<{ name: string; version: string }, 200>
+  H.HttpResponse<ApplicationInfo, 200> | H.HttpResponse<unknown, 500>,
+  Dependencies
 > = H.of((_: H.HttpRequest) =>
-  RTE.of(
-    H.successJson({
-      name: getValueFromPackageJson("name"),
-      version: getCurrentBackendVersion(),
-    }),
+  pipe(
+    RTE.ask<Dependencies>(),
+    RTE.chain(({ InfoService, PackageUtils }) =>
+      RTE.fromTaskEither(InfoService.getPackageInfo({ PackageUtils })),
+    ),
+    RTE.map((info) => H.successJson(info)),
+    RTE.mapLeft((_problems) => new H.HttpError()),
```

Comment:
If I remember correctly, all errors that we want to manage needs to be remapped to a RTE right, otherwise the framework will always output a 500 as if an unmanaged error occurred. do you agree?

cc. @gquadrati
```suggestion
    RTE.mapLeft((_problems) => new H.HttpError()),
    RTE.orElseW((error) =>
      RTE.right(H.problemJson({ status: error.status, title: error.message })),
    ),
```

Reply:
Commit f2cae0e78fe0b03e6e7c59b5b01cdbde960c79c8 contains an update to the error test case